middleware/has-artwork.ts

```diff
@@ -0,0 +1,12 @@
+import { useArtworkStore } from "@/stores/artworks";
+
```

Comment:
Ik denk dat de naam van de middleware de lading niet dekt.
Ook zou ik hier graag een kleine comment zien waarvoor deze middleware dient.
Ik meen dat dit ervoor moet zorgen dat deeplinks naar artworks redirecten naar de corresponderende category als het artwork nog niet in de store staat. Dit om extra logica voor het afhandelen van deze use-case te voorkomen.